deployment/dynatrace-azure-logs.sh

```diff
@@ -412,6 +423,36 @@ else
     done
     echo ""
 
+    echo "Do you want to apply Azure tags to new created resources?"
+    echo "Y - Yes"
+    echo "N - No"
+    while ! [[ "${APPLY_TAGS}" =~ ^(Y|N)$ ]]; do
+        read -p "Apply Azure tags?: " -i N -e APPLY_TAGS
+    done
+    echo ""
+
+    case ${APPLY_TAGS} in
+    Y)
+        APPLY_TAGS=true
+        ;;
+    N)
+        APPLY_TAGS=false
+        ;;
+    *)
+        echo "\e[91mERROR: \e[37mAPPLY_TAGS - unexpected option value"
+        exit 1
+        ;;
+    esac
+
+    if [[ "${APPLY_TAGS}" == "true" ]]
```

Comment:
please use here bash convention as you have in another places like in lines 429 and 450:
```
if [[ "${APPLY_TAGS}" == "true" ]]; then
```